### PR TITLE
add job priority for kube-batch scheduling

### PR DESCRIPTION
**Pull request by 4everming**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/mpi-operator/141)

### CODE REVIEW

**Comment by 4everming** [comment]
@googlebot I signed it!

**Comment by xieydd** [comment]
@4everming How about fixing the CI error.

**Comment by 4everming** [comment]
> @4everming How about fixing the CI error.

Got it.

File: deploy/crd/crd-v1alpha2.yaml [hunk 1]
```diff
@@ -23,6 +23,8 @@ spec:
             slotsPerWorker:
               type: integer
               minimum: 1
+            jobPriority:
+              type: string
```
Comment (rongou) [inline deploy/crd/crd-v1alpha2.yaml]: Why is this a string? Shouldn't it be an integer?

Comment (4everming) [reply deploy/crd/crd-v1alpha2.yaml]: Hi rongou, jobPriority refers to PriorityClassName just like "system-node-critical" or any other priority.  That is consistent with PriorityClassName in PodGroupSpec of kube-batch scheduling.

**Comment by terrytangyuan** [comment]
@4everming Should this be done in kubeflow/common instead so that If there are more parameters like this in the future for scheduling, we don’t have to keep adding fields to MPIJobSpec. Related discussion: https://github.com/kubeflow/common/issues/11

**Comment by terrytangyuan** [comment]
Closing this since the changes have been submitted to kubeflow/common.

**Pull request closed without merging**